### DISCUSSION TITLE
Individual event color customization

### DISCUSSION THREAD

**Discussion by konata-san** [davish/obsidian-full-calendar#414] [Feature Requests]
It would be nice to be able to modify the color of individual events within a calendar. This would make it much easier to categorize different types of events without having to make a whole new calendar for each category.

**Comment by gerroon** [comment]
Hi
This would be so  amazing, everything has the same color now in my calender :(

**Comment by YouFoundJK** [comment]
Checkout the [advanced categorization](https://youfoundjk.github.io/plugin-full-calendar/events/categories/) in my fork [here](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest).

<img width="1350" height="891" alt="FUllCalendar" src="https://github.com/user-attachments/assets/4cb711fd-41ff-4480-8b21-4d93cc82942a" />
